LICENSE.txt

```diff
@@ -0,0 +1,215 @@
+====
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+====
```

Comment:
this seems like an uncommon solution. It doesn't look right. Perhaps this is to make a license checker to pass?

Reply:
Yes, It is generated by `mvn license:format` to make  license checker pass.